### DISCUSSION TITLE
Mac OSX Sonoma 15.4 / M1 Max problem

### DISCUSSION THREAD

**Discussion by byAdamGrzesik** [Azure-Samples/graphrag-accelerator#50] [Q&A] [answered]
Hi, 
I have tried to deploy the solution (cloned into the docker container), and this error pops up:

Creating container registry... ERROR: rosetta error: failed to open elf at /lib64/ld-linux-x86-64.so.2

Any ideas anyone?

**Comment by byAdamGrzesik** [comment, marked as answer]
Don't use container. It works as pure cloned repository.

**Reply by timothymeyers** [reply]
Not the ideal solution but I'm glad you got it working! 

Looks like this is an issue with the Azure CLI / Bicep running inside devcontainers on Apple silicon. https://github.com/Azure/azure-cli/issues/25726

I'm running the devcontainer inside a GitHub Codespace streamed locally to my MBP running Sonoma 14.5 / M3 Max, for what it's worth.
